src/libs/ReportUtils.ts

```diff
@@ -1603,6 +1630,7 @@ function getIcons(
             name: personalDetails?.[report?.ownerAccountID ?? -1]?.displayName ?? '',
             fallbackIcon: personalDetails?.[report?.ownerAccountID ?? -1]?.fallbackIcon,
         };
+        // eslint-disable-next-line @typescript-eslint/no-shadow
         const isPayer = currentUserAccountID === report?.managerID;
 
         return isPayer ? [managerIcon, ownerIcon] : [ownerIcon, managerIcon];
```

Comment:
Let's rename the variable here to also avoid the no-shadow eslint error, also the managerID is not always the payer, so the correct variable name here would be `isManager`:

```suggestion
        const isManager = currentUserAccountID === report?.managerID;
        return isManager ? [managerIcon, ownerIcon] : [ownerIcon, managerIcon];
```